packages/scaffold/src/views/w3m-wallet-send-preview-view/index.ts

```diff
@@ -114,11 +130,64 @@ export class W3mWalletSendPreviewView extends LitElement {
     return null
   }
 
-  private onSendClick() {
-    RouterController.reset('Account')
-    setTimeout(() => {
-      SendController.resetSend()
-    }, 200)
+  private async onSendClick() {
```

Comment:
I think moving business logic to the controllers might help us to keep the UI components and partials smaller. Since we are also using different controllers inside the function, this might also decrease the dependencies of these components. This would be a function on SendController which gets only parameters and does the things. Wdyt?

Reply:
I agree, added business logic to `SendController`